docs/en/contributing/install.md

```diff
@@ -11,7 +11,7 @@ Clone the repository, `cd` into the project, install [Lerna](https://github.com/
 ```shell
 git clone https://github.com/Microsoft/fast-dna.git
 cd fast-dna
-npm i --global lerna@3.3.0
+npm i --global lerna
```

Comment:
the correct version is installed on `npm i` and it's not necessary to explicitly call out the version that will frequently change over time and could grow out of sync.

Reply:
Can we stop from pinning during the install in the build process then as well?

Reply:
Can we update the build server as well then?

Reply:
Yep.